coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -182,6 +182,140 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
 
   private val queryParams = PromQlQueryParams("notUsedQuery", 100, 1, 1000)
 
+  it("Plan with unary expression should be equals to its binary counterpart.") {
```

Comment:
Can you add one test in this file for something like this?
```
-(foo{} + bar{})
```
Just to triple-check that a root-level unary operator is applied as-expected.